<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputDialogConfirmacao</class>
 <widget class="QWidget" name="InputDialogConfirmacao">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1032</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Informações</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="labelAviso">
     <property name="text">
      <string>Se for o caso, marcar os itens com problema.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="frameData">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="labelEvento">
        <property name="text">
         <string>Data evento:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDateEdit" name="dateEditEvento">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="TableView" name="tableLogistica">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSalvar">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Salvar</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBoxQuebrado">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="labelFoto_2">
        <property name="text">
         <string>Foto quebrado:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditFoto_2"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonFoto_2">
        <property name="text">
         <string>Selecionar...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonQuebradoEntrega">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Quebrado/Faltando</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonQuebradoReceb">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Quebrado/Faltando</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QGroupBox" name="">
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditEntregou"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelEntregou">
        <property name="text">
         <string>Quem entregou:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelRecebeu">
        <property name="text">
         <string>Quem recebeu:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditRecebeu"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelFoto">
        <property name="text">
         <string>Foto entrega:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditFoto"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButtonFoto">
        <property name="text">
         <string>Selecionar...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>src/tableview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>dateEditEvento</tabstop>
  <tabstop>tableLogistica</tabstop>
  <tabstop>pushButtonQuebradoEntrega</tabstop>
  <tabstop>pushButtonQuebradoReceb</tabstop>
  <tabstop>lineEditFoto</tabstop>
  <tabstop>pushButtonFoto</tabstop>
  <tabstop>lineEditRecebeu</tabstop>
  <tabstop>lineEditEntregou</tabstop>
  <tabstop>pushButtonSalvar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
